starters/next12-routing-chakra-ui/README.md

```diff
@@ -0,0 +1,183 @@
+# nextjs12-routing-chakra-ui starter kit
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [CLI (Recommended)](#cli-recommended)
+  - [Manual](#manual)
+- [Commands](#commands)
+- [State management](#state-management)
+- [Testing](#testing)
+  - [Unit testing](#unit-testing)
+  - [Storybook's Accessibility testing](#storybooks-accessibility-testing)
+- [Chakra UI's Customizable theme](#chakra-uis-customizable-theme)
+- [Example Components](#example-components)
+  - [API routes example](#api-routes-example)
+  - [Counter Example](#counter-example)
+  - [Fetch example](#fetch-example)
+- [Kit Organization / Architecture](#kit-organization--architecture)
+- [How to deploy your project](#how-to-deploy-your-project)
+
+## Overview
+
+This Next.js 12 starter kit comes with formatting, linting, example components, testing and styling. Here is the complete list of technologies and tooling used in this kit:
+
+- [Next.js 12](https://nextjs.org/blog/next-12)
+- [Chakra UI](https://chakra-ui.com/)
+- [Jest](https://jestjs.io/) - Test runner
+- [TypeScript](https://www.typescriptlang.org/) - Type checking
+- [Storybook](https://storybook.js.org/) - Component library
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Installation
+
+### CLI (Recommended)
+
+```bash
+npm create @this-dot/starter --kit next12-routing-chakra-ui
```

Comment:
```suggestion
npm create @this-dot/starter -- --kit next12-routing-chakra-ui
```

I'm pretty sure the npm version of this command needs the extra -- to pass the args off to the create starter cli